src/core/dbus-cgroup.c

```diff
@@ -602,8 +602,12 @@ int bus_cgroup_set_property(
 
         flags |= UNIT_PRIVATE;
 
-        if (streq(name, "CPUAccounting"))
-                return bus_cgroup_set_boolean(u, name, &c->cpu_accounting, CGROUP_MASK_CPUACCT|CGROUP_MASK_CPU, message, flags, error);
+        if (streq(name, "CPUAccounting")) {
+                CGroupMask mask = CGROUP_MASK_CPUACCT;
+                if (need_cpu_controller_for_accounting())
```

Comment:
I'm not sure if this is correct, since this might not be local, right? If this isn't correct for that reason I'll just remove it and only set `CGROUP_MASK_CPUACCT`.

Reply:
why not use the cached version here?